src/v/coproc/wasm_event.cc

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright 2020 Vectorized, Inc.
+ *
+ * Licensed as a Redpanda Enterprise file under the Redpanda Community
+ * License (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * https://github.com/vectorizedio/redpanda/blob/master/licenses/rcl.md
+ */
+
+#include "coproc/wasm_event.h"
+
+#include "bytes/iobuf_parser.h"
+#include "coproc/errc.h"
+#include "coproc/logger.h"
+#include "hashing/secure.h"
+#include "vlog.h"
+
+#include <array>
+namespace coproc {
+
+std::string_view action_as_string_view(wasm_event_action action) {
+    static constexpr std::string_view deploy{"deploy"};
+    static constexpr std::string_view remove{"remove"};
+    switch (action) {
+    case wasm_event_action::deploy:
+        return deploy;
+    case wasm_event_action::remove:
+        return remove;
+    }
+    __builtin_unreachable();
+}
+
+std::string_view header_as_string_view(wasm_event_header header) {
+    static constexpr std::string_view action{"action"};
+    static constexpr std::string_view description{"description"};
+    static constexpr std::string_view checksum{"checksum"};
```

Comment:
remember to change `checksum` for `sha256` or whatever we decide to use, which allows for hashing evolution. i.e.: tomorrow we might change it for a cheaper or more expensive one.